app/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "com.parasoft.demoapp"
+        minSdk 16
+        targetSdk 32
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    sourceSets {
+        main {
+            res.srcDirs = [
+                    // Add layout files path
```

Comment:
Is there specific reason why we put both parent and children directory here? 
Because according to the [best practice](https://developer.android.com/studio/build/build-variants#configure-sourcesets) here, seems that should be avoided.
![Capture](https://user-images.githubusercontent.com/35082985/184757717-43473b7c-70cd-4221-806f-87772995a268.PNG)